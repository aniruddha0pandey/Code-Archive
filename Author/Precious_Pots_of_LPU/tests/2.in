3
44 4
26 45 93 67 54 86 43 78 61 74 70 32 34 85 77 7 59 24 8 98 79 35 11 14 4 66 33 10 52 94 37 41 60 76 50 46 15 48 22 100 97 25 62 9 
93 79 10 15 
40 4
96 75 98 41 64 38 52 42 79 12 3 85 43 14 50 5 15 29 68 57 8 59 72 67 9 24 53 36 33 18 48 62 2 100 92 26 6 99 16 70 
3 59 33 2 
41 2
11 27 57 17 30 95 61 41 1 71 91 14 93 56 36 87 97 69 16 3 94 44 26 15 47 8 2 74 96 58 38 85 34 7 28 89 77 39 99 50 54 
71 58 
