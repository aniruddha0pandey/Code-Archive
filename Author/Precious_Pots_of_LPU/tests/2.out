26 45 67 54 86 43 78 61 74 70 32 34 85 77 7 59 24 8 98 35 11 14 4 66 33 52 94 37 41 60 76 50 46 48 22 100 97 25 62 9 93 79 10 15 
96 75 98 41 64 38 52 42 79 12 85 43 14 50 5 15 29 68 57 8 72 67 9 24 53 36 18 48 62 100 92 26 6 99 16 70 3 59 33 2 
11 27 57 17 30 95 61 41 1 91 14 93 56 36 87 97 69 16 3 94 44 26 15 47 8 2 74 96 38 85 34 7 28 89 77 39 99 50 54 71 58 
